samples/cocoapods/kotlin-library/build.gradle.kts

```diff
@@ -0,0 +1,37 @@
+plugins {
+    id("org.jetbrains.kotlin.multiplatform")
+    id("org.jetbrains.kotlin.native.cocoapods")
+}
+
+repositories {
+    jcenter()
+    maven { setUrl("https://dl.bintray.com/kotlin/kotlin-dev") }
+    maven { setUrl("https://dl.bintray.com/kotlin/kotlin-eap") }
+}
+
+group = "org.jetbrains.kotlin.sample.native"
+version = "1.0"
+
+kotlin {
+    // Add a platform switching to have an IDE support.
+    val buildForDevice = System.getenv("SDK_NAME")?.startsWith("iphoneos") ?: false
```

Comment:
I find it kinda more reliable to check some specific build environment like presence of `kotlin.native.cocoapods.target` project property.

Reply:
I mean that we can and should select between device and simulator basing on `SDK_NAME` value, but I would prefer selecting between IDE and non-IDE to be based on `kotlin.native.cocoapods.target` if possible.

Reply:
Yes, it makes sense. Fixed.